addons/udes_security/security/ir.model.access.csv

```diff
@@ -1,3 +1,4 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 udes_allowed_file_type,udes_allowed_file_type,model_udes_allowed_file_type,base.group_user,1,0,0,0
 access_udes_readonly_desktop_user_model_exception,access_udes_readonly_desktop_user_model_exception,model_udes_readonly_desktop_user_model_exception,base.group_user,1,0,0,0
+access_res_archiving_restrictions,access_res_archiving_restrictions,model_res_archiving_restrictions,base.group_no_one,1,1,1,1
```

Comment:
Suggestion: Add a readonly permission for all users for safety, like the readonly model exception.